posting.go

```diff
@@ -111,6 +111,18 @@ type PostingsList struct {
 // represents an immutable, empty postings list
 var emptyPostingsList = &PostingsList{}
 
+func (i *PostingsList) ResetBytesRead(uint64) {}
```

Comment:
Add some commentary here, that is NOT implemented for v11, v12, v13, v14.

Reply:
Done, thanks